linkerd/app/core/src/admin/mod.rs

```diff
@@ -158,23 +168,11 @@ fn rsp(status: StatusCode, body: impl Into<Body>) -> Response<Body> {
         .expect("builder with known status code must not fail")
 }
 
-fn check_loopback<B>(req: &Request<B>) -> Result<(), Response<Body>> {
-    if let Some(addr) = req.extensions().get::<ClientAddr>() {
-        let addr = addr.as_ref();
-        if addr.ip().is_loopback() {
-            return Ok(());
-        }
-        tracing::warn!(%addr, "denying request from non-loopback IP");
-        Err(rsp(
-            StatusCode::FORBIDDEN,
-            "access to /proxy-log-level and /trace only allowed from loopback interface",
-        ))
```

Comment:
any reason we no longer return Forbidden? 404 seems much less semantically correct in this particular case. not a big deal though.